### PR TITLE
Sidebox list

### PR DESCRIPTION
**Pull request by wiztechinc**
Edits to code and text as well as improved information sidebox image.

### CODE REVIEW

**Comment by wiztechinc** [comment]
as I said, I can't view the floats from the new css. I made sure the code was the same thru out but I see some spacing issues that may or may not show.

**Comment by scottcwilson** [comment]
This is a good first effort.  You can see the results here: 

https://deploy-preview-412--zencartdocs.netlify.app/user/sideboxes/sidebox_list/

Thing that need to be done: 

- you have some lingering list <ul>, etc markup.  Please remove.

- You have elements that are not closed and that messes things up.  Look at the sentence starting with "This can be found in the setting" and see how the <em> is not closed.

Please do those changes, then submit them with this PR using : 

git commit --amend 
git push --force 

that way the PR will only show one commit.

**Comment by wiztechinc** [comment]
Looks like a couple of edits did not make it into the final file.  I don't think github desktop allows the amend/force option.  Alternatives?

**Comment by scottcwilson** [comment]
no worries, I will squash your commits.  Please proceed.

**Comment by wiztechinc** [comment]
the order history text isn't floating. My code problem or image size problem?

File: content/user/sideboxes/sidebox_list.md [hunk 1]
```diff
@@ -158,7 +158,6 @@ This can be found in the setting <em>Admin > Configuration > Layout Settings</em
 <br clear="all">
 <br>
 
-
 # Languages
```
Comment (scottcwilson) [inline content/user/sideboxes/sidebox_list.md:161]: Please use `##` like the other headers  do for this and subsequent headers.

File: content/user/sideboxes/sidebox_list.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ This can be found in the setting <em>Admin > Configuration > Layout Settings</em
       <br><br>
       There are two more banner sideboxes, <strong>banner_box2</strong> and <strong>banner_box_all</strong>, which are configured similarly. <strong>Banner_box2</strong> is just a duplicate of the first, but <strong>banner_box_all</strong> is different in that it can display multiple banners randomly. 
    </div>
+	</div>
 <br clear="all">
 
 <ul>
```
Comment (scottcwilson) [inline content/user/sideboxes/sidebox_list.md:29]: Get rid of this `<ul>`.  Look for others and get rid of them too - also `</ul>` and `<li>`, etc.

Comment (scottcwilson) [reply content/user/sideboxes/sidebox_list.md:29]: You will want to use a plain text editor to find issues like this.   An HTML or Markdown editor might hide them from you.

**Comment by wiztechinc** [comment]
The unstyled div is only around the img_col and notes_col?  Why is it even necessary? I was confused by its presence. Sorry about the first commit. Keeping hugo and github docs separate for now and didn't copy over the final file before the commit. 

I truly am uncertain about lists - I copied that from your original work, I thought. Are those additional boxes supposed to be in an unordered list or not?

**Comment by scottcwilson** [comment]
Please get rid of the list artifacts.  Just leave the unstyled div for now so we can get this finished.

**Comment by wiztechinc** [comment]
My apologies, so you mean by artifacts all lists? I am deficient in code speak as I am entirely self-taught and I am unsure exactly what you mean and that is why I asked. Easy to do, harder to restore.

**Comment by scottcwilson** [comment]
There are a bunch of `<ul>` and `<li>` (and associated closing tags) that are remnants from when this document was entirely text with no images.  Please remove them; they are no longer necessary.  The new structure of the document is header followed by div block containing text and image.  No list at all.

**Comment by scottcwilson** [comment]
> Easy to do, harder to restore.

In a year or less, you won't believe you ever thought this.  Source control is a superpower.

**Comment by wiztechinc** [comment]
No sweat. Already feeling it in the past...ish. Confidence restored! Commit coming up.

**Comment by scottcwilson** [comment]
OK close enough.  I will fix the remaining issues.

**Comment by wiztechinc** [comment]
glad to see the back of that one. I should ask what the remaining issues were but I'm toasted.

**Comment by scottcwilson** [comment]
Take a look tomorrow at #415.  (Ignore the other file, just look at the changes to sidebox_list.md)

Thanks for taking this on - the page looks much better now!

**Comment by wiztechinc** [comment]
Whew!

**Pull request merged**